### PR TITLE
Update DG, UG, AboutMe

### PR DESCRIPTION
**Pull request by boundtotheearth**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -285,6 +285,21 @@ _{More to be added}_
 
 **Extensions**
 
+1.  User requests to start a timer for a specific project in the list
+2.  Momemtum starts the timer for the project
+3.  User works on their project for an amount of time
+4.  User requests to end a timer for a specific project in the list
+5.  Momemtum ends the timer for the project
+6.  Momentum displays the time period that the timer ran for
```
Comment (pr4aveen) [inline docs/DeveloperGuide.md]: Should these sentences end with a fullstop

**Review by pr4aveen — APPROVED** [review]
One issue with consistency. Should the sentences in each bullet point end with a fullstop?

LGTM otherwise

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (boundtotheearth) [reply docs/DeveloperGuide.md]: Ok fixed

**Review by khoodehui — APPROVED** [review]
LGTM

**Pull request merged**